src/main/groovy/org/ajoberstar/grgit/auth/AuthConfig.groovy

```diff
@@ -97,11 +99,13 @@ class AuthConfig {
     static final String FORCE_OPTION = 'org.ajoberstar.grgit.auth.force'
     static final String USERNAME_OPTION = 'org.ajoberstar.grgit.auth.username'
     static final String PASSWORD_OPTION = 'org.ajoberstar.grgit.auth.password'
+	static final String PASSPHRASE_OPTION = 'org.ajoberstar.grgit.auth.passphrase'
```

Comment:
Maybe making the password parallel to this private key path property would be helpful. Something like this?

`org.ajoberstar.grgit.auth.ssh.passphrase`